fix(nlu): update NLU slot type to be a map with custom property names (#91)
spokestack/react-native-spokestack: README.md

• **intent**: string

_Defined in [src/types.ts:98](https://github.com/spokestack/react-native-spokestack/blob/ddbb73c/src/types.ts#L98)_

The intent based on the match provided by the NLU model

---

##### confidence

• **confidence**: number

_Defined in [src/types.ts:100](https://github.com/spokestack/react-native-spokestack/blob/ddbb73c/src/types.ts#L100)_

> A percentage of the confidence of the match, given by the tensorflow

Reviewer: ```suggestion
A percentage representing the NLU model's confidence in the intent it recognized
```

> A percentage of the confidence of the match, given by the tensorflow

Reviewer 2: ```suggestion
A decimal `[0,1]` representing the NLU model's confidence in the intent it recognized, where 1 represents absolute confidence.
```

---

##### slots

• **slots**: { [key:string]: SpokestackNLUSlot; }

_Defined in [src/types.ts:102](https://github.com/spokestack/react-native-spokestack/blob/ddbb73c/src/types.ts#L102)_

Data associated with the intent, provided by the NLU model

---

#### SpokestackNLUSlot

##### type

• **type**: string

_Defined in [src/types.ts:91](https://github.com/spokestack/react-native-spokestack/blob/ddbb73c/src/types.ts#L91)_

> 

Reviewer: ```suggestion

The slot's type, as defined in the model metadata

```

---

##### value

• **value**: string

_Defined in [src/types.ts:92](https://github.com/spokestack/react-native-spokestack/blob/ddbb73c/src/types.ts#L92)_

> 

Reviewer: ```suggestion

The parsed (typed) value of the slot

```

---

##### rawValue

• **rawValue**: string

> 

Reviewer: ```suggestion

The original string value of the slot recognized in the user utterance

```
